lib/ui/notifications.dart

```diff
@@ -16,48 +17,182 @@ GlobalConfiguration cfg = GlobalConfiguration();
 
 class NotificationsWidgetState extends State<NotificationsWidget> {
   GlobalConfiguration cfg = GlobalConfiguration();
+  var notificationSubList = <NotificationSubscription>[];
+  var notificationSub = new NotificationSubscription('Test Keyword');
+  //var notificationsList = <Notification>[];
+  //var notification = new Notification(DateTime.now(), '', '', '');
+
 
   @override
   void initState() {
     super.initState();
   }
 
+  void addItemToList(){
+    setState(() {
+      notificationSubList.add(notificationSub);
+
+      //notificationsList.add(notification);
+    });
+  }
+
+  void removeItemFromList(String item) {
+    setState(() {
+      var itemindexSubList = notificationSubList.indexWhere((element) => element.keyword == item);
+      //var itemindexNList = notificationsList.indexWhere((element) => element.subscriptionKeyword == item);
+      notificationSubList.removeAt(itemindexSubList);
```

Comment:
Instead of using the index, check out `removeWhere`.
See: https://api.flutter.dev/flutter/dart-core/List/removeWhere.html

Reply:
Jack, just checking.  But, is this what an implementation of removeWhere should look like notificationSubList.removeWhere((element) => element.keyword == item);  and if so its going to remove all instances of the same word correct?  I just want to make sure my test is working correctly because right now debug is using "Test Keyword" and the above code is clearing the entire screen.